src/parse-config/parse-config.ts

```diff
@@ -112,8 +114,15 @@ class ConfigParser {
       await this.fetchDataForEntity(entityPath);
     }
 
-    if (typeof value === "string" && value.startsWith("$fn")) {
-      value = myEval(value.slice(3));
+    if (typeof value === "string") {
+      if (typeof value === "string" && value.startsWith("$ex")) {
```

Comment:
Why do you check again for the type of `value` ?

Reply:
To be extra sure?
Thanks, I fixed that now.